modules/packages/VisualDebug.chpl

```diff
@@ -130,50 +132,28 @@ private proc VDebugTree (what: vis_op, name: string, time: real, id: int = 0,
   :arg tagname: name of the tag
 */
   proc tagVdebug ( tagname : string ) {
-    //coforall l in Locales[1..] do
-    //  on l do chpl_vdebug_tag (tagname.localize().c_str(), 0);
-    //chpl_vdebug_tag (tagname.localize().c_str(), 0);
     if (VisualDebugOn) {
-       VDebugTree (vis_op.v_tag, tagname, 0);
+       chpl_vdebug_tagname (tagname.c_str(), tagno);
```

Comment:
That is a good question.   tagVdebug() is supposed to be called only on locale-0.    Having two different tagVdebug() calls happening at the same time from two or more tasks would also cause problems for chplvis in trying to figure out the data.   I would have no problem making sure this code runs uninterrupted.